src/dht_proxy_server.cpp

```diff
@@ -129,6 +132,13 @@ DhtProxyServer::DhtProxyServer(std::shared_ptr<DhtRunner> dht, in_port_t port ,
                 scheduler_.edit(printStatsJob_, scheduler_.time() + PRINT_STATS_PERIOD);
             }
         });
+        getNodeStatsJob_ = scheduler_.add(scheduler_.time(), [this]{
+            if (service_->is_up() and not stopListeners) {
```

Comment:
service doesn't need to be up, might this allow to update the stats during startup with this job ?

Reply:
Bad copy paste from the previous job. But indeed no need to check this here

Reply:
Done